mysql-test/suite/rocksdb_rpl/t/optimize_myrocks_replace_info.test

```diff
@@ -0,0 +1,29 @@
+--source include/have_rocksdb.inc
+--source include/master-slave.inc
+--source include/have_binlog_format_row.inc
+
+connection master;
+
+create table t1(c1 int,c2 int, primary key (c1)) engine=rocksdb;
+insert into t1 values(1,1),(2,2),(3,3);
+select * from t1;
+
+
+
+replace into t1 values(1,11);
```

Comment:
Could you add another test case about locking --  1. "replace into t1 values(1,11);" blocks updates/deletes to the same key (1) until a transaction is committed, and 2. replace is blocked if there is already a running transaction locking the same key?

Reply:
OK